content/sensu-core/2.0/reference/agent.md

```diff
@@ -0,0 +1,803 @@
+---
+title: "Agent Reference"
+linkTitle: "Agent"
+description: "Sensu agent reference documentation"
+weight: 1
+version: "2.0"
+product: "Sensu Core"
+platformContent: true
+platforms: ["Linux", "Windows"]
+menu:
+  sensu-core-2.0:
+    parent: reference
+---
+
+- [Installation][1]
+- [Creating events using service checks](#creating-events-using-service-checks)
+- [Creating events using the agent socket](#creating-events-using-the-agent-socket)
+- [Creating events using the StatsD listener](#creating-events-using-the-statsd-listener)
+- [Keepalive monitoring](#keepalive-monitoring)
+- [Operation](#operation)
+	- [Starting the agent](#starting-the-service)
```

Comment:
In #818, the back-end is referred to as the service as that is the topic of the document. Should that remain consistent here instead of using `agent`?